release/RELEASE.md

```diff
@@ -70,8 +72,8 @@ our luck with a random new one.
 
 1. Run through the following test plan on one of Linux or MacOS:
 
-   1. Install the new SDK using `curl -sSL https://get.daml.com/ | sh -s X.XX.XX`,
-      where `X.XX.XX` is the new version number.
+   1. Install the new SDK using `curl -sSL https://get.daml.com/ | sh -s $(cat
+      LATEST | awk '{print $2}')`.
```

Comment:
Perhaps `cut -d' ' -f2 < LATEST` instead of `awk`?

Either way, we should probably add either `coreutils` or `gawk` to the list of Nix dependencies.

Reply:
It may just be an upbringing difference, but I find `awk` more "natural" and `print $2` more explicit that w're printing the second field. Good point on adding to dev-env.

Reply:
Turns out `gawk` is already in dev-env. I should just use that then.

Reply:
Fair enough. I prefer `awk` too, mostly because `cut -d' '` treats every space as a delimiter, but I'm told it's more confusing. Happy to stick with `awk`.